android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -1559,7 +1559,8 @@ public static boolean registerLexicalModel(HashMap<String, String> lexicalModelI
     // Disable sugestions if lexical-model file doesn't exist
     if (!modelFile.exists()) {
       modelFileExists = false;
-      setBannerOptions(false);
+      setBannerOptions(false, KeyboardType.KEYBOARD_TYPE_INAPP);
+      setBannerOptions(false, KeyboardType.KEYBOARD_TYPE_SYSTEM);
```

Comment:
The language is the same on INAPP and SYSTEM keyboard, so update banner options for both keyboards.